code/game/objects/items/cards_ids.dm

```diff
@@ -111,8 +113,8 @@
 	registered_account.replaceable = TRUE
 
 	// Applying the trim updates the label and icon, so don't do this twice.
-	if(ispath(trim))
-		SSid_access.apply_trim_to_card(src, trim)
+	if(ispath(trim_path) && !trim)
+		SSid_access.apply_trim_to_card(src, trim_path)
```

Comment:
I'm guessing this runtime that I've been seeing on sybil is what this fixes? Been eyeing it for a few days

```
[2022-08-12 09:53:52.419] runtime error: Cannot read /datum/id_trim/job/prisoner (/datum/id_trim/job/prisoner).sechud_icon_state
 - proc name: get trim sechud icon state (/obj/item/card/id/proc/get_trim_sechud_icon_state)
 -   source file: cards_ids.dm,790
 -   usr: Helvetica8563 (/mob/dead/new_player)
 -   src: the prisoner ID card (/obj/item/card/id/advanced/prisoner)
 -   usr.loc: the plating (8,174,1) (/turf/open/floor/plating)
 -   src.loc: Haliee Mason (/mob/living/carbon/human/dummy)
 -   call stack:
 - the prisoner ID card (/obj/item/card/id/advanced/prisoner): get trim sechud icon state()
 - the prisoner ID card (/obj/item/card/id/advanced/prisoner): get trim sechud icon state()
 - the prisoner ID card (/obj/item/card/id/advanced/prisoner): get sechud job icon state()
 - Haliee Mason (/mob/living/carbon/human/dummy): sec hud set ID()
 - Haliee Mason (/mob/living/carbon/human/dummy): equip to slot(the prisoner ID card (/obj/item/card/id/advanced/prisoner), 256, 1, 0)
 - Haliee Mason (/mob/living/carbon/human/dummy): equip to slot if possible(the prisoner ID card (/obj/item/card/id/advanced/prisoner), 256, 1, 1, 0, 1, 1)
 - Haliee Mason (/mob/living/carbon/human/dummy): equip to slot or del(the prisoner ID card (/obj/item/card/id/advanced/prisoner), 256, 1)
 - Prisoner (/datum/outfit/job/prisoner): equip(Haliee Mason (/mob/living/carbon/human/dummy), 1)
 - Haliee Mason (/mob/living/carbon/human/dummy): equipOutfit(/datum/outfit/job/prisoner (/datum/outfit/job/prisoner), 1)
 - Haliee Mason (/mob/living/carbon/human/dummy): dress up as job(/datum/job/prisoner (/datum/job/prisoner), 1)
 - ...
 - /datum/preferences (/datum/preferences): create character preview view(Helvetica8563 (/mob/dead/new_player))
 - /datum/preferences (/datum/preferences): ui data(Helvetica8563 (/mob/dead/new_player))
 - /datum/tgui (/datum/tgui): get payload(null, 1, 1)
 - /datum/tgui (/datum/tgui): open()
 - /datum/preferences (/datum/preferences): ui interact(Helvetica8563 (/mob/dead/new_player), /datum/tgui (/datum/tgui))
 -  (/atom/movable/screen/lobby/button/character_setup): Click(null, "mapwindow.map", "icon-x=61;icon-y=21;left=1;but...")
 -  (/atom/movable/screen/lobby/button/character_setup):  Click(null, "mapwindow.map", "icon-x=61;icon-y=21;left=1;but...")
 - /datum/callback/verb_callback (/datum/callback/verb_callback): Invoke()
 - /datum/callback/verb_callback (/datum/callback/verb_callback): InvokeAsync()
 - 
```

Reply:
Likely yes. trim stored both the path, and the object created from the path. the bug there could just be trying to get icon state before the trim is new'd, but I'm not sure when that ordinarily happens